src/Microsoft.ServiceHub.Framework/ServiceHubHostRemoteServiceBroker.cs

```diff
@@ -11,8 +11,16 @@ namespace Microsoft.ServiceHub.Framework;
 /// ServiceBroker provided to services running inside of ServiceHub Hosts. Wraps an existing <see cref="RemoteServiceBroker"/>
 /// and adds the <see cref="ServiceActivationOptions"/> ServiceHubHostProcessId to each request.
 /// </summary>
+[Obsolete("This API may soon be moved.")]
```

Comment:
Yes. This is from when we were playing around with the idea of allowing ServiceHub Host processes to run as low integrity. The process id that we inject into the activation options was used to determine the elevation status of the host process IIRC.

Reply:
Great. Deleted.

Reply:
So... you said I could delete this, but then in #15 you brought back a dependency on this class. So I have to undelete it. Does that sound right, or is the code you brought back not supposed to depend on it? Maybe I can at least just mark it `internal`.

Reply:
We can delete the usage of it from there. We'll also need to update the DevCore and Gladstone repos to remove the other usages. It won't be a breaking change because no one is using the ActivationArgument that its injecting into the service requests anymore.

Reply:
Alright. Can you carefully review [the last commit I just pushed](https://github.com/microsoft/vs-servicehub/pull/16/commits/9d4475b149687b99ee757a9df52c52738590299c)  to see that it matches what you hand in mind?

Reply:
Not exactly. I can make the changes in your branch.

Reply:
Pushed the changes